rollup.config.js

```diff
@@ -3,10 +3,10 @@ import dts from "rollup-plugin-dts";
 
 export default [
   {
-    input: "index.ts",
+    input: "index.mts",
     output: {
       dir: "dist",
-      format: "module",
+      format: "es",
```

Comment:
This cleans up the syntax. `module` is not mentioned as a valid option for 'format' in the rollup docs.  `es` looks like the right thing and since it is the default, I bet rollup was falling back to `es`.